WORKSPACE.bazel

```diff
@@ -4,8 +4,8 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
     
 http_archive(
     name = "rules_swiftnav",
-    strip_prefix = "rules_swiftnav-e79fa5d8d8cbb19974877397bf045822af9a5125",
-    url = "https://github.com/swift-nav/rules_swiftnav/archive/e79fa5d8d8cbb19974877397bf045822af9a5125.tar.gz",
+    strip_prefix = "rules_swiftnav-70823deec53c70820bffc3ae4cb295c88bbdd735",
```

Comment:
Needs to be updated to non-branch commit before merging.

Reply:
thanks!